lib/src/scryfall.rs

```diff
@@ -267,46 +267,58 @@ lazy_static! {
             "Desolate Lighthouse",
             ManaCost::from_rgbuwc(0, 0, 0, 0, 0, 1)
         ),
+        (
+            "Command Tower",
+            ManaCost::from_rgbuwc(1, 1, 1, 1, 1, 0)
+        ),
+        (
+            "Opal Palace",
+            ManaCost::from_rgbuwc(1, 1, 1, 1, 1, 1)
+        ),
+        (
+            "Path of Ancestry",
+            ManaCost::from_rgbuwc(1, 1, 1, 1, 1, 0)
+        ),
         // fetch lands
         (
             "Arid Mesa",
-            ManaCost::from_rgbuwc(1, 0, 0, 0, 1, 0)
+            ManaCost::from_rgbuwc(1, 1, 1, 1, 1, 0)
```

Comment:
I'm not sure I understand why the Zendikar fetch lands should count for more than their two mana color pips. For example, https://gatherer.wizards.com/pages/card/details.aspx?multiverseid=177584 should only count as red and white?

Reply:
You can find every dual land, shockland, snow dual either mountain or plains and so on for every fetchland.